api/events/schema.py

```diff
@@ -104,3 +136,13 @@ class Mutation(graphene.ObjectType):
             [required]\n- start_time: The start time of the calendar event\
             [required]\n- end_time: The field with the end time of the calendar\
              event[required]")
+    end_event = EndEvent.Field(
+        description="Mutation to end a calendar event given the arguments\
+            \n- calendar_id: The unique identifier of the calendar event\
+            [required]\n- event_id: The unique identifier of the target\
```

Comment:
Good work @ThaDeveloper . However, consider deleting one of the duplicate `event_id` field description

Reply:
@TeamoreA This has been worked on.